### DISCUSSION TITLE
Busco equipo para el hackathon!

### DISCUSSION THREAD

**Discussion by Sebastian-Longhitano** [Launch-X-Latam/MisionFrontEnd#546] [Team Posts]
Aún me encuentro en la búsqueda de equipo!
Pueden contactarme por éste medio!

lx14356@innovaccion.mx

Gracias!

**Comment by MarisaE-S** [comment]
Hola yo también busco ***@***.***

**Comment by mafer23** [comment]
https://discord.gg/QZa2UAHu
